goal_poses:
  - {x: -4.38975, y: -1.60161, theta: -34.14118}
  - {x: -4.73559, y: 1.11929, theta: 80.66521}
  - {x: 2.22491, y: 2.41651, theta: -76.65672}
  - {x: 1.13148, y: -0.54157, theta: 104.49494}

pose_number: 4

position_tolerance: 0.3 # 位置误差容忍阈值（米）

angle_tolerance_deg: 15.0 # 角度误差容忍阈值（度）

loop_navigation: true  # 设置为true启用循环导航，设置为false禁用循环

